### ISSUE TITLE
Tablet Theme

### ISSUE THREAD

**Issue by Xitro01** [matt8707/hass-config#52]
Hello @matt8707,

Really love what you did with this dashboard, have looked at many alternatives, but this one is by far one of the sleekest designs I've seen. Therefore I would like to take the leap and use this dashboard and enjoy the many hours of customization.

But until that's ready, I would like to use my old dashboard/theme. I have already put this one next to it, by adding the following configuration:

```
 lovelace:
   mode: storage
   dashboards:
     lovelace-yaml:
       mode: yaml
       title: Dashboard
       icon: mdi:tablet-dashboard
       show_in_sidebar: true
       filename: ui-lovelace.yaml
```

The only thing that bugs me is that I can't apply your tablet-theme only on the ui-lovelace.yaml. I've added "theme: tablet" there, but it doesn't seem to work. Only when I change the complete backend theme of HomeAssistant it seems to work, but this will override my old dashboard's theme as well.

**TL;DR**
Is there any way to apply your tablet-theme on your ui-lovelace.yaml dashboard only, without having to set it as the global theme?

Thanks in advance.

**Comment by matt8707** [comment]
Hi, unfortunately no

https://github.com/thomasloven/lovelace-layout-card/issues/21#issuecomment-502011210
